From d41045a1d11856a6b38284786e66b1a651f585f2 Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sony.com>
Date: Fri, 22 Jun 2018 20:24:43 +0900
Subject: [PATCH] Replace hardcoded build barier with a generic one

Bootctrl is used when the when the AB OTA is enabled
We can replace the hardcoded platform filter with the
generic AB OTA flag.

Test: manual - build ok. device boots ok.

Change-Id: I34ee90a2818ad23cc6b9233bdde126a0965fae0d
Signed-off-by: Alin Jerpelea <alin.jerpelea@sony.com>
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 03a11d3..bb91f07 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,6 +1,6 @@
 # TODO:  Find a better way to separate build configs for ADP vs non-ADP devices
 ifneq ($(BOARD_IS_AUTOMOTIVE),true)
-ifneq ($(filter msm8996 msm8998 sdm845,$(TARGET_BOARD_PLATFORM)),)
+ifeq ($(AB_OTA_UPDATER),true)
 LOCAL_PATH := $(call my-dir)
 
 # HAL Shared library for the target. Used by libhardware.
-- 
2.13.0

